joint_1_controller:
  type: "position_controllers/JointPositionController"
  joint: joint_1

# joint_1_controller:
#   type: "velocity_controllers/JointVelocityController"
#   joint: "joint_1"
#   pid: 
#     p: 100.0
#     i: 0.1
#     d: 0.01

joint_2_controller:
  type: "position_controllers/JointPositionController"
  joint: joint_2

joint_3_controller:
  type: "position_controllers/JointPositionController"
  joint: joint_3

joint_4_controller:
  type: "position_controllers/JointPositionController"
  joint: joint_4

